### PR TITLE
Add keybase script for arbitrary futzing

### PR DESCRIPTION
**Pull request by mlsteele**
`keybase script <scriptname> [<args>...]` is for running arbitrary junk in the service for debugging. Not compiled into prod builds.

`keybase script rid2 gui rook1@rooter` runs `ResolveThenIdentify2` and pretty prints the results.

### CODE REVIEW

File: go/service/debugging_production.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+import (
+	"fmt"
+
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+	"golang.org/x/net/context"
+)
+
+func (t *DebuggingHandler) Script(ctx context.Context, arg keybase1.ScriptArg) (string, error) {
+	defer t.G().CTraceTimed(ctx, "Script", func() error { return err })()
```
Comment (zanderz) [inline go/service/debugging_production.go]: err is undefined here, this broke production

Comment (mlsteele) [reply go/service/debugging_production.go]: argh sorry. I'll be more careful. But also any chance CI can compile production?

**Comment by zanderz** [comment]
NBD, stuff happens.
I kind of think one daily sanity prod build would be OK - otherwise we'd be bulking up CI a lot, wouldn't we? Not sure of the best way to cover that.

**Comment by mlsteele** [comment]
yeah hopefully it won't come up again. Fixed in master now

**Pull request merged**